### PR TITLE
Provide consul health check status as label

### PR DESCRIPTION
**Pull request by avthart**
As discussed in #52 and #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ __What service nodes are failing?__
 
     sum by (node, service)(consul_catalog_service_node_healthy == 0)
 
+__What service checks are critical?__
+	
+	consul_health_service_status{status="critical"} == 1
```
Comment (grobie) [inline README.md]: Nitpick: Let's use whitespace consistently, it seems the other code examples use 4 spaces instead one tab.

**Comment by avthart** [comment]
@grobie Thank you again for your feedback. I have just added health maintenance state and converted tab to spaces in README.md

**Comment by grobie** [comment]
Looks great, thank you! :+1:

**Comment by avthart** [comment]
Thanks!

Op do 11 mei 2017 om 08:37 schreef Tobias Schmidt <notifications@github.com>

**Pull request merged**